src/components/TextField/TextField.tsx

```diff
@@ -430,8 +430,10 @@ export function TextField({
     'aria-activedescendant': ariaActiveDescendant,
     'aria-autocomplete': ariaAutocomplete,
     'aria-controls': ariaControls,
-    'aria-multiline': normalizeAriaMultiline(multiline),
     'aria-expanded': ariaExpanded,
+    ...(multiline && {
+      'aria-multiline': normalizeAriaMultiline(multiline),
+    }),
```

Comment:
I did something similar in my a11y checks PR but got a comment from @BPScott and Andrew asking if we could just change the return type in the normalize function instead of the spread / check combo. 

```ts
function normalizeAriaMultiline(multiline?: boolean | number) {
  if (!multiline) return undefined;

  return Boolean(multiline) || multiline > 0
    ? {'aria-multiline': true}
    : undefined;
}
```

Then you could replace these lines with 

```suggestion
    ...normalizeAriaMultiline(multiline),
```

Reply:
There are also two tests that would have to be fixed because they're expecting to receive `aria-multiline="false"` which is incorrect

Reply:
Ahhh that is awesome, I will do that. Thanks @kyledurand ✨